pkg/pgmodel/delete/delete.go

```diff
@@ -79,7 +78,7 @@ func getMetricNameSeriesIDFromMatchers(conn pgxconn.PgxConn, matchers []*labels.
 func filterMetricNameSeriesIds(metrics, schemas []string, seriesIds [][]model.SeriesID) (filteredMetrics []string, filteredSeriesIds [][]model.SeriesID) {
 	for i := range metrics {
 		metricSchema := schemas[i]
-		if metricSchema != schema.Data {
+		if metricSchema != "prom_data" {
```

Comment:
I think we should leave the schema package alone and this reverts as it was. Don't want to litter the code with hardcoded schema names like this.

Reply:
I think I got all of these.

Reply:
I also renamed the constant names to be in line with their contents.